### ISSUE TITLE
Got errors when running npm install mediasoup --save

### ISSUE THREAD

**Issue by Gogistics** [ibc/mediasoup#52]
I got errors below when running **npm install mediasoup --save**

> 6293 error Linux 4.8.6-x86_64-linode78
6294 error argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "mediasoup" "--save"
6295 error node v7.4.0
6296 error npm  v4.0.5
6297 error code ELIFECYCLE
6298 error mediasoup@0.3.0-pre3 postinstall: `make Release`
6298 error Exit status 2
6299 error Failed at the mediasoup@0.3.0-pre3 postinstall script 'make Release'.
6299 error Make sure you have the latest version of node.js and npm installed.
6299 error If you do, this is most likely a problem with the mediasoup package,
6299 error not with npm itself.
6299 error Tell the author that this fails on your system:
6299 error     make Release
6299 error You can get information on how to open an issue for this project with:
6299 error     npm bugs mediasoup
6299 error Or if that isn't available, you can get their info via:
6299 error     npm owner ls mediasoup
6299 error There is likely additional logging output above.

**Comment by ibc** [comment]
It's hard to help if you don't provide more info regarding operating system, etc. Also, do you have mediasoup requirements installed?: https://mediasoup.org/overview/#requirements

**Comment by ibc** [comment]
Just tested in a fresh Debian server with same Node version installed. It works. Closing issue, please reopen if you can provide more info and you have the requirements installed.

**Comment by Gogistics** [comment]
@ibc thank you for the prompt reply. Yes, I did installed all requirements. Let me try again.

My Linux Distribution is: Debian 8
node: v7.4.0
npm: v4.0.5
Python 2.7.9
gcc version 4.9.2
GNU Make 4.0

**Comment by Gogistics** [comment]
@ibc  I found the issue which is g++ not found. I have to do **apt-get install build-essential**

**Comment by ibc** [comment]
Thanks. I will update the website to make it clear that a C++ compiler is required.

**Comment by ibc** [comment]
Done.

**Comment by Gogistics** [comment]
Yes, Thank you very much. Next, I'm going to build a simple app to test the server.

**Comment by nguyenthuynhatanh** [comment]
![image](https://user-images.githubusercontent.com/29257126/35025423-08f6d7f8-fb78-11e7-9299-5208d6189d4c.png) 
it don't have anything in web demo and it different with your demo
